composer.lock

```diff
@@ -4,7 +4,7 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "d91b6375bf70c3bb6848882f9867eb94",
```

Comment:
Obviously, this file needs to be restored to match the corresponding reversion of `composer.json`.